.gitignore

```diff
@@ -21,6 +21,7 @@
 *.profclang?
 *.profraw
 *.dyn
+*.ipynb
```

Comment:
Please move this to your `core.excludesfile` file.

Reply:
OK, I removed it.